src/app/(content-pages)/transform-property/TransformPropertyPage.tsx

```diff
@@ -33,12 +33,18 @@ export default function TransformPropertyPage() {
       <div className="grid grid-cols-1 md:grid-cols-1 gap-6 md:gap-10">
         <Image
           src={beforeAfter}
-          alt={
-            "A Philadelphia lot before a clean up and the same lot, filled with trees and greenery after a clean up."
-          }
+          alt=""
           placeholder="blur"
           className="w-full overflow-hidden rounded-[20px] aspect-video md:aspect-auto object-cover object-center"
+          aria-labelledby="caption-before-after-image"
         />
+        <h2
```

Comment:
The caption doesn't look or act like a heading, so let's just change it to a p element. That might fix your "gap" issue as well.

Reply:
@zhenbright Heads up in case you missed this comment, let's update this h2 to a p element, and fix the spacing.

Reply:
Hi @zhenbright can you address the PR feedback? Thanks!